modules/monitoring-expiring-silences.adoc

```diff
@@ -16,22 +16,22 @@ Expired silences older than 120 hours are garbage collected.
 
 .Procedure
 
-To expire a silence in the *Administrator* perspective:
+To expire a silence or silences in the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Silences* page.
+. Go to *Observe* -> *Alerting* -> *Silences*.
 
-. For the silence you want to modify, select the {kebab} in the last column and choose *Expire silence*.
+. For the silence or silences you want to expire, select the checkbox in the corresponding row.
+
+. Click *Expire silence*.
```

Comment:
see picture below, if you select 1 silence, the button is "Expire 1 silence", 
![Screenshot from 2023-09-01 08-51-22](https://github.com/juzhao/test/assets/22743651/68bf345c-3fd9-4fc0-baae-1b665d231d28)
select 2 silences, the button is "Expire 2 silences", and so on
![Screenshot from 2023-09-01 08-53-24](https://github.com/juzhao/test/assets/22743651/6864c6b0-d606-4be4-a147-0c62c40534e2)